extension-testcontainer/src/main/java/io.camunda.zeebe.process.test.extension.testcontainer/ContainerizedEngine.java

```diff
@@ -120,13 +120,17 @@ public void increaseTime(final Duration timeToAdd) {
   }
 
   @Override
-  public long getEngineTime() {
+  public long getTime() {
     final ManagedChannel channel = getChannel();
-    final EngineControlBlockingStub stub = getStub(channel);
+    try {
+      final EngineControlBlockingStub stub = getStub(channel);
 
-    final GetEngineTimeRequest request = GetEngineTimeRequest.newBuilder().build();
-    final GetEngineTimeResponse response = stub.getEngineTime(request);
-    return response.getCurrentTime();
+      final GetTimeRequest request = GetTimeRequest.newBuilder().build();
+      final GetTimeResponse response = stub.getTime(request);
+      return response.getTime();
+    } finally {
+      closeChannel(channel);
+    }
```

Comment:
🔧 We could solve this without a try-finally block:
```java
    final EngineControlBlockingStub stub = getStub(channel);

    final GetTimeRequest request = GetTimeRequest.newBuilder().build();
    final GetTimeResponse response = stub.getTime(request);
    final long time = response.getTime();
    closeChannel(channel);
    return time;
```

Reply:
If something happens calling the gRPC stub and it throws an exception, wouldn't you leak the channel?

Reply:
That's a fair point. I'll have to change this for some of the other methods as well. Thanks!